x-pack/packages/kbn-elastic-assistant/impl/data_anonymization_editor/stats/anonymized_stat/index.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiStat, EuiText, EuiToolTip } from '@elastic/eui';
+import React, { useMemo } from 'react';
+// eslint-disable-next-line @kbn/eslint/module_migration
+import styled from 'styled-components';
+
+import { getColor, getTooltipContent } from './helpers';
+import { ANONYMIZATION_ICON, TITLE_SIZE } from '../helpers';
+import * as i18n from './translations';
+
+const AnonymizationIconFlexItem = styled(EuiFlexItem)`
+  margin-right: ${({ theme }) => theme.eui.euiSizeS};
```

Comment:
Note for me to update the Discover/O11y patch as euiTheme didn't work w/ them (probably where I put the integration in the tree...)